### DISCUSSION TITLE
File uploads get a 403 forbidden

### DISCUSSION THREAD

**Discussion by iiDestiny** [spatie/laravel-google-cloud-storage#44] [Q&A]
## version

```
 "spatie/laravel-google-cloud-storage": "^1.0",
 "laravel/framework": "^8.0",
```

## config

![image](https://user-images.githubusercontent.com/17637481/191305598-f039b60b-ba5b-4b44-ae0f-c9bdb3149ad5.png)

![image](https://user-images.githubusercontent.com/17637481/191305744-0f69943d-7222-4fb5-881b-b428f1b53e93.png)

## code

![image](https://user-images.githubusercontent.com/17637481/191305824-9811700e-1a52-4f38-a09a-a7614db7ccf1.png)

## Get Feedback

![image](https://user-images.githubusercontent.com/17637481/191305343-51c5b8ea-4114-4d58-947e-9a0918396818.png)

**Comment by iiDestiny** [comment]
@hotrush @freekmurze   thanks

**Comment by hotrush** [comment]
Hey

```
 "spatie/laravel-google-cloud-storage": "^1.0",
```

When the latest release is 2.x. But anyways it looks like wrongly configured Iam permissions (according to error message)
